L/libpointmatcher/bundled/patches/comment-cmake-alias.patch

```diff
@@ -0,0 +1,13 @@
+diff --git before/CMakeLists.txt after/CMakeLists.txt
```

Comment:
Are you sure this patch is really needed?

Reply:
I am not very confident in this patch. This was the only way I could get the library to reliably build during the wizard, so if we can get rid of it, I am all for it.